make/tools.mk

```diff
@@ -47,7 +47,7 @@ TOOLS += ko=v0.13.0
 # Version of Gateway API install bundle https://gateway-api.sigs.k8s.io/v1alpha2/guides/#installing-gateway-api
 GATEWAY_API_VERSION=v0.6.2
```

Comment:
I can't remember why this is still on `v0.6.2`, can it be updated to `v0.7.1` @irbekrm ?

Reply:
I can't remember a particular reason and having had a brief look at the release notes I can't see anything potentially breaking, so should be ok to update it.